system/lib/compiler-rt/emscripten_setjmp.c

```diff
@@ -56,3 +59,8 @@ uint32_t testSetjmp(uint32_t id, TableEntry* table, uint32_t size) {
   }
   return 0;
 }
+
+void emscripten_longjmp(uintptr_t env, int val) {
```

Comment:
Cool... I tried this but I got some odd crashed in the compiler.. but if this works that is great!